<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>881</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="but_StartStop">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>40</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Start / Stop update</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_UpdateFrequency">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_Averaging">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_NumDatapoints">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>10</y>
      <width>70</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <item>
     <property name="text">
      <string>20</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>50</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1000</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>10</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>QBPM readout frequency (Hz):</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>QBPM value averaging:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Number of plotted values:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Update status:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_update">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>40</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>inactive</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Display upper limit:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_UpperLimit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="maxVisibleItems">
     <number>20</number>
    </property>
    <item>
     <property name="text">
      <string>auto</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1.0e-5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6.0e-6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2.5e-6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1.0e-6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6.0e-7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2.5e-7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1.0e-7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6.0e-8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2.5e-8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1.0e-8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6.0e-9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2.5e-9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1.0e-9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6.0e-10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2.5e-10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1.0e-10</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_LowerLimit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>67</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="maxVisibleItems">
     <number>20</number>
    </property>
    <item>
     <property name="text">
      <string>auto</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1.0e-5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6.0e-6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2.5e-6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1.0e-6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6.0e-7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2.5e-7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1.0e-7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6.0e-8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2.5e-8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1.0e-8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6.0e-9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2.5e-9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1.0e-9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6.0e-10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2.5e-10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1.0e-10</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>67</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Display lower limit:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>67</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PETRA beam current:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_p3current">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>67</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>inactive</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
